apps/token/src/routes/staking/shared.ts

```diff
@@ -13,6 +14,67 @@ export const rawValidatorScore = (
     : null;
 };
 
-export const normalisedVotingPower = (votingPower: string) => {
+export const getNormalisedVotingPower = (votingPower: string) => {
   return toBigNum(votingPower, 0).dividedBy(100).dp(2).toString() + '%';
 };
+
+export const getUnnormalisedVotingPower = (
+  validatorScore: string | null | undefined
+) =>
+  validatorScore
+    ? new BigNumber(validatorScore).times(100).dp(2).toString() + '%'
+    : null;
+
+export const getFormattedPerformanceScore = (performanceScore: string) =>
+  new BigNumber(performanceScore).dp(2);
+
+export const getPerformancePenalty = (performanceScore: string) =>
+  new BigNumber(1)
+    .minus(getFormattedPerformanceScore(performanceScore))
+    .times(100)
+    .toString() + '%';
```

Comment:
instead of adding a percent to strings we should be using the formatPercentage helper (which I think exists. If it doesn't then it should)